Patch generated 20071214 by Robby Workman <rworkman@slackbuilds.org>
Permission granted to use this patch as you see fit.

--- FreeImage/Makefile.fip.orig	2007-12-14 13:53:23.424826247 -0600
+++ FreeImage/Makefile.fip	2007-12-14 14:21:10.427625070 -0600
@@ -14,13 +14,13 @@
 # Converts cr/lf to just lf
 DOS2UNIX = dos2unix
 
-COMPILERFLAGS = -O3 -fexceptions -fvisibility=hidden
+COMPILERFLAGS = -fexceptions -fvisibility=hidden
 LIBRARIES = -lstdc++
 
 MODULES = $(SRCS:.c=.o)
 MODULES := $(MODULES:.cpp=.o)
-CFLAGS = $(COMPILERFLAGS) $(INCLUDE)
-CXXFLAGS = $(COMPILERFLAGS) -Wno-ctor-dtor-privacy $(INCLUDE)
+CFLAGS += $(COMPILERFLAGS) $(INCLUDE)
+CXXFLAGS += $(COMPILERFLAGS) -Wno-ctor-dtor-privacy $(INCLUDE)
 
 TARGET  = freeimageplus
 STATICLIB = lib$(TARGET).a
@@ -59,13 +59,14 @@
 	$(CC) -s -shared -Wl,-soname,$(VERLIBNAME) -o $@ $(MODULES) $(LIBRARIES)
 
 install:
-	install -m 644 -o root -g root $(HEADER) $(INCDIR)
-	install -m 644 -o root -g root $(HEADERFIP) $(INCDIR)
-	install -m 644 -o root -g root $(STATICLIB) $(INSTALLDIR)
-	install -m 755 -o root -g root $(SHAREDLIB) $(INSTALLDIR)
-	ln -sf $(SHAREDLIB) $(INSTALLDIR)/$(VERLIBNAME)
-	ln -sf $(VERLIBNAME) $(INSTALLDIR)/$(LIBNAME)
-	ldconfig
+	install -d 755 $(DESTDIR)/$(INCDIR)
+	install -m 644 $(HEADER) $(DESTDIR)/$(INCDIR)
+	install -m 644 $(HEADERFIP) $(DESTDIR)/$(INCDIR)
+	install -d 755 $(DESTDIR)/$(INSTALLDIR)
+	install -m 644 $(STATICLIB) $(DESTDIR)/$(INSTALLDIR)
+	install -m 755 $(SHAREDLIB) $(DESTDIR)/$(INSTALLDIR)
+	cd $(DESTDIR)/$(INSTALLDIR) ; ln -sf $(SHAREDLIB) $(VERLIBNAME)
+	cd $(DESTDIR)/$(INSTALLDIR) ; ln -sf $(VERLIBNAME) $(LIBNAME)
 
 clean:
 	rm -f core Dist/*.* u2dtmp* $(MODULES) $(STATICLIB) $(SHAREDLIB) $(LIBNAME)
